src/main/java/com/github/maracas/roseau/diff/changes/BreakingChangeKind.java

```diff
@@ -29,7 +29,175 @@ public enum BreakingChangeKind {
 	TYPE_FORMAL_TYPE_PARAMETERS_ADDED(MUTATION),
 	TYPE_FORMAL_TYPE_PARAMETERS_REMOVED(MUTATION),
 	TYPE_FORMAL_TYPE_PARAMETERS_CHANGED(MUTATION),
-
+	/**
+	 * A breaking change where a method has been removed or its accessibility has
+	 * been reduced.
+	 * <p>
+	 * This occurs when a method that was present in the old version of a library is
+	 * either:
+	 * <p>
```

Comment:
You can use a `ul` `li` syntax here (see https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#examples)